scripts/manifest.js

```diff
@@ -44,10 +45,20 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.resolve(__dirname, '../svg/color/', fileName);
+	let color;
+	if (fs.existsSync(svgPath)) {
+		const svg = fs.readFileSync(svgPath, 'utf8');
+		const colors = getColors(svg);
+		const hex = colors.fills[0].hex().toUpperCase();
+		color = hex;
```

Comment:
I think we should add a check here so it doesn't error if for some reason there is no color.

Also, no need to convert to uppercase, according to the docs it already is: https://github.com/colorjs/get-svg-colors#usage

You also don't need to save each state into a variable, it could just be expressed as:

```js
const fillColor = getColors(svg).fills[0];
color = fillColor ? fillColor.hex() : undefined;
```

Reply:
There's a mistake in the docs, it actually returns a lowercase hex if you don't call ```.toUpperCase``` on it